src/css/base.css

```diff
@@ -17,6 +17,7 @@
 
   --tablet-container-max-width: 704px;
   --desktop-container-max-width: 1088px;
+  --container-max-width: 335px;

```

Comment:
As paddings is 20px, should be 375

Reply:
--tablet-container-max-width: 768px;
--desktop-container-max-width: 1088px;
--container-max-width: 375px;

I think that should to fit the Figma.